src/components/todoFooter.js

```diff
@@ -18,9 +19,30 @@ export default class TodoFooter extends React.Component {
 					<strong>{todoModel.activeTodoCount}</strong> {activeTodoWord} left
 				</span>
 				<ul className="filters">
-						{this.renderFilterLink(ViewModel.ALL_TODOS, "", "All")}
-						{this.renderFilterLink(ViewModel.ACTIVE_TODOS, "active", "Active")}
-						{this.renderFilterLink(ViewModel.COMPLETED_TODOS, "completed", "Completed")}
+					<li>
+						<RefreshLink
+							navigationData={{filter: ViewModel.ALL_TODOS}}
+							stateNavigator={stateNavigator}
+							activeCssClass="selected">
+							All
+						</RefreshLink>
+					</li>
+					<li>
+						<RefreshLink
+							navigationData={{filter: ViewModel.ACTIVE_TODOS}}
+							stateNavigator={stateNavigator}
+							activeCssClass="selected">
+							Active
+						</RefreshLink>
+					</li>
+					<li>
+						<RefreshLink
+							navigationData={{filter: ViewModel.COMPLETED_TODOS}}
+							stateNavigator={stateNavigator}
+							activeCssClass="selected">
+							Completed
+						</RefreshLink>
+					</li>
```

Comment:
Builds Hyperlinks using the `RefreshLink` component. Instead of hard-coding Urls, specifies the new data in the props. The `RefreshLinks` output the 'selected' class name when they're active.